App.ts

```diff
@@ -25,7 +29,8 @@ class App {
     this.routes();
```

Comment:
Maybe moving this line after initializing all other variables is probably a better choice!